### PR TITLE
k6: Add version 0.25.1

### LINKED ISSUE #3220: [Request] k6.io - (Another) load-testing tool
K6 is a developer-centric load testing tool, written in GO and Javascript; 

Binaries are available here:
https://github.com/loadimpact/k6/releases

(They also provide MSI installers here: https://docs.k6.io/docs/installation)

### PR DESCRIPTION
**Pull request by LoZeno**
Adding k6.io, resolves #3220

~~It's my first time creating a scoop manifest, I'm not sure how to test the autoupdate~~ nvm, one needs to always read the whole wiki before opening their mouth. Autoupdate tested and works.

### CODE REVIEW

File: bucket/k6.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+    "homepage": "https://k6.io/",
+    "description": "K6: developer centric open source load and performance regression testing tool",
```
Comment (Ash258) [inline bucket/k6.json]:
```suggestion
    "description": "Load and performance regression testing tool for cloud native backend infrastructure",
```

File: bucket/k6.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+    "homepage": "https://k6.io/",
```
Comment (Ash258) [inline bucket/k6.json]:
```suggestion
    "homepage": "https://k6.io",
```

File: bucket/k6.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            "extract_dir": "k6-v0.25.1-win64"
+        },
+        "32bit": {
+            "url": "https://github.com/loadimpact/k6/releases/download/v0.25.1/k6-v0.25.1-win32.zip",
+            "hash": "6776890125143bf9d9bea97597230601a581cccd2be7be0f97cfae2422519d65",
+            "extract_dir": "k6-v0.25.1-win32"
+        }
+    },
+    "bin": "k6.exe",
+    "shortcuts": [
```
Comment (Ash258) [inline bucket/k6.json]: image

**Comment by LoZeno** [comment]
Moved to [Main](https://github.com/ScoopInstaller/Main/pull/593) as requested

**Pull request closed without merging**